tasks/typedoc.ts

```diff
@@ -0,0 +1,97 @@
+import ITask = grunt.task.ITask;
```

Comment:
We could submit a PR, but this seems to be a pretty common practice in DefinitelyTyped (around 450 packages prefix interface names with I). It might be more worthwhile to submit a discussion issue to DT asking if the no-I style (and possibly other styles in the [TS Coding Guidelines](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines)) should become a general recommendation.